Applications/FileIO/GocadIO/GocadTSurfaceReader.h

```diff
@@ -0,0 +1,126 @@
+/**
+ *
+ * @copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/LICENSE.txt
+ */
+
+#pragma once
+
+#include <iostream>
```

Comment:
include iosfwd in the header file, if possible, an only the needed stream headers in the cpp.

Reply:
:heavy_check_mark: